db/warehouse/migrate/20210527140359_add_head_of_household_for_to_hap_client.rb

```diff
@@ -0,0 +1,5 @@
+class AddHeadOfHouseholdForToHapClient < ActiveRecord::Migration[5.2]
+  def change
+    add_column :hap_report_clients, :head_of_household_for, :string, array: true
```

Comment:
not jsonb?

Reply:
I started down the route of using 'ANY' which seemed simpler with arrays.